backend/src/packages/users/user.repository.ts

```diff
@@ -44,6 +44,32 @@ class UserRepository implements Repository {
     });
   }
 
+  public async findByRoleKey(
+    key: 'chatbot' | 'user',
```

Comment:
let's create a enum for this
```suggestion
    key: ValueOf<typeof UserRoleKey>,
```